Assets/Scripts/Player/PlayerController.cs

```diff
@@ -147,19 +153,32 @@ private void OnRightClick(InputAction.CallbackContext obj)
             if (Physics.Raycast(clickRay, out RaycastHit hit, 10000) && _attackLogic.Status == AttackLogic.AttackStatus.None)
             {
                 GameObject objectHit = hit.collider.gameObject;
+                Interactable interactable = hit.collider.GetComponent<Interactable>();
                 IDamageable damageable = objectHit.GetComponent<IDamageable>();
+
                 if (damageable != null)
                 {
                     // implementation NOT capable of area damage
```

Comment:
Don't forget to RemoveFocus here :) or was it on purpose?

Reply:
Oh yeah i want to also focus on the damageable as well, so addition to attack we can do something more if we attach Interactable.cs on it. Without Interactable.cs nothing will be focused.

Reply:
I think you misunderstood me, @Kaes3kuch3n updated the code